src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,134 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final Integer VALID_AGE = 20;
+    private static final String INVALID_LOGIN = "v";
+    private static final String INVALID_PASSWORD = "p";
+    private static final Integer INVALID_AGE = 10;
+    private static final String NOT_EXISTING_LOGIN = "notExisting";
+    private static final int PASSWORD_MINIMAL_LENGTH = 6;
+    private static final int LOGIN_MINIMAL_LENGTH = 6;
+    private static final int USER_MINIMAL_AGE = 18;
+    private static StorageDaoImpl storage;
```

Comment:
```suggestion
    private static StorageDao storage;
```
use abstract types for variable types